code/headers/lidar.hpp

```diff
@@ -1,118 +0,0 @@
-#pragma once
```

Comment:
Why is this file deleted?

Reply:
This is in lidar.hpp in the feature branch of the sonar module, not sure how this happened

Reply:
I returned the file and corrected the outdated usart.